.tekton/README.md

```
@@ -0,0 +1,222 @@
+environment. Pipelines are triggered by
+[Pipelines-as-Code](https://pipelinesascode.com/) based on repository events
+and comments.
+
+For Tekton concepts (Tasks, Pipelines, PipelineRuns), see the
+[Tekton Getting Started](http://tekton.dev/docs/getting-started/) documentation.
+
+---
+
+## Overview
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)